static/js/components/RunSummary.jsx

```diff
@@ -0,0 +1,428 @@
+          }
+        },
+        {
+          name: "Status",
+          options: {
+            filter: true
+          }
+        },
+        {
+          name: "RA",
+          options: {
+            filter: false,
+            customBodyRenderLite: renderRA
+          }
+        },
+        {
+          name: "Dec",
+          options: {
+            filter: false,
+            customBodyRenderLite: renderDec
+          }
+        },
+        {
+          name: "Redshift",
+          options: {
+            filter: false
+          }
+        },
+        {
+          name: "Requester",
+          options: {
+            filter: true
+          }
+        },
+        {
+          name: "Request",
+          options: {
+            filter: true
+          }
+        },
+        {
+          name: "Priority",
+          options: {
+            filter: true
+          }
+        },
+        {
+          name: "Rise Time (UT)",
+          options: {
+            filter: false,
+            customBodyRenderLite: (
+              (dataIndex) => new Date(assignments[dataIndex].rise_time_utc).toLocaleTimeString()
+            )
+          }
+        },
+        {
+          name: "Set Time (UT)",
+          options: {
+            filter: false,
+            customBodyRenderLite: (
+              (dataIndex) => new Date(assignments[dataIndex].set_time_utc).toLocaleTimeString()
+            )
+          }
+        },
+        {
+          name: "Finder",
+          options: {
+            filter: false,
+            customBodyRenderLite: renderFinderButton
+          }
+        },
+        {
+          name: "Actions",
+          options: {
+            filter: false,
+            customBodyRenderLite: renderActionsButton
+          }
+        }
+      ];
+
+
+    const options = {
+      draggableColumns: { enabled: true },
+      expandableRows: true,
+      renderExpandableRow: renderPullOutRow,
+      selectableRows: "none"
+    };
+
+    const data = observingRun.assignments.map(
+      (assignment) => ([
+        assignment.obj.id,
+        assignment.status,
+        assignment.obj.ra,
+        assignment.obj.dec,
+        assignment.obj.redshift,
+        assignment.requester.username,
+        assignment.comment,
+        assignment.priority,
+        assignment.rise_time_utc,
+        assignment.set_time_utc,
+        null,
+        null])
+    );
+
+    return (
+      <div className={styles.source}>
+        <div>
+          <Grid container direction="column" alignItems="center" justify="flex-start" spacing={3}>
+            <Grid item>
+              <div>
+                <Typography variant="h4" gutterBottom color="textSecondary" align="center">
+                  <em>Observing Planner for:</em>
+                </Typography>
+                <Typography variant="h4" gutterBottom color="textSecondary">
+                  <b>{observingRunTitle(observingRun, instrumentList, telescopeList, groups)}</b>
```

Comment:
Make into a React component

Reply:
`observingRunTitle` just returns a string -- is it really necessary to make that a reactComponent?

Reply:
Ah, no in that case it's fine. Disregard.